boards/arm/mimxrt1050_evk/doc/mimxrt1050_evk.rst

```diff
@@ -26,13 +26,17 @@ MIMXRT1050-EVKB board, refer to `Board Revisions`_ section.
 Hardware
 ********
 
-- MIMXRT1052DVL6A MCU (600 MHz, 512 KB TCM)
+- MIMXRT1052DVL6A MCU (Cortex-M7, 600 MHz, 32KB ICache/32KB DCache, MPU,
+  VFPv5 FPU, Crypto Engine)
 
 - Memory
 
-  - 256 KB SDRAM
-  - 64 Mbit QSPI Flash
-  - 512 Mbit Hyper Flash
+  - 512 KByte MCU-internal SRAM
+  - 32 MByte external SDRAM (166MHz)
+  - 8 MByte external QSPI Flash
+  - 64 MByte external Hyper Flash
+  - 96 KByte MCU BootROM for loading application from external memory
```

Comment:
I prefer "KB" and "MB", but will defer to @dbkinder

Reply:
my preference too (KB, MB)